crates/cheatnet/src/forking/state.rs

```diff
@@ -68,7 +68,7 @@ fn other_provider_error<T>(boxed: impl ToString) -> Result<T, StateError> {
     let err_str = boxed.to_string();
 
     Err(StateReadError(
-        if err_str.contains("error sending request for url") {
+        if err_str.contains("error sending request") {
```

Comment:
note: Seems that error message was reduced, with the old `error sending request for url`, test `test_calling_nonexistent_url ` is failing. See https://github.com/foundry-rs/starknet-foundry/actions/runs/15439759021/job/43454570622?pr=3416

Reply:
> It might be from reqwest or some other network related package maybe?

That's what I've mentioned 😄  _It's probably from one of blockifier's request/network-related dependencies_